app/redux/selectors/SetsSelectors.js

```diff
@@ -259,4 +261,50 @@ export const hasChangesToSync = (state) => {
     return (root.setIDsToUpload.length > 0 || root.setIDsBeingUploaded.length > 0);
 };
 
+// collapsed metrics
+
+export const getBestAvgVelocityEver = (state, set) => {
+    let historySets = getHistorySetsChronological(state);
```

Comment:
- [ ]  don't use getHistorySetsChronological, make a getHistorySets that converts it but doesn't sort it